src/e2e/README.md

```diff
@@ -0,0 +1,49 @@
+# How to Run e2e Test
+
+###
+
+#### Install npm packages error will show if node 18.12.0^ is not installed, `nvm install 18.12.0` should fix this.
+
+```sh
+npm install
+```
+
+#### Install playwright along with necessary browser agents
+
+```sh
+npx playwright install
+```
+
+#### Generate .env from .env-dist
+
+```sh
+cp .env-dist .env
+```
+
+#### Add some additional values to the .env file
+
+`E2E_TEST_ACCOUNT_EMAIL=`
+
+`E2E_TEST_ACCOUNT_PASSWORD=`
+
+`E2E_TEST_BASE_URL=`
+
+`E2E_TEST_ENV=` if this is not set, stage will be used
```

Comment:
I don't see this key in the current .env-dist file, we might want to add it and possibly default it to something (or leave a comment on expected/valid values):

https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/.env-dist#L131-L134

I was able to get a rough hint by searching for keys in the repo and finding this, but not sure what the allowed values are:
https://github.com/mozilla/blurts-server/blob/fe9cafba924c924deb1c1146ce08b33d0916f228/.github/workflows/playwright.yml#L78

Reply:
good point, updating the .env-dist to include this